scss/components/calendar.scss

```diff
@@ -166,6 +168,7 @@ $fd-calendar-table-width: $fd-calendar-table-item-width * 7 !default;
     flex: 1;
     justify-content: space-between;
     align-items: center;
+    padding: 12px;
```

Comment:
_Seems_ wrong.  If padding is desired within the popover, it should be the popover that contains the padding (or a child of it that is a parent to the entire calendar component).

Reply:
@greg-a-smith Many components are using the popover. Applying padding to the popover would lead to undesired results to the other components using it.

Reply:
I agree, it shouldn't be default padding on the popover itself, but some sort of calendar wrapper when it is using within the popover to get the appropriate padding that wouldn't exist on a standalone calendar.

Reply:
I believe the fd-calendar is the wrapper which contains all the elements of a calendar. In the newest version of this file, the padding are used match the latest design sheet.